bindings/node.js/lib/kzg.js

```diff
@@ -31,19 +53,49 @@ function transformTrustedSetupJson(filePath) {
   return outputPath;
 }
 
-const originalLoadTrustedSetup = bindings.loadTrustedSetup;
-// docstring in ./kzg.d.ts with exported definition
-bindings.loadTrustedSetup = function loadTrustedSetup(filePath) {
-  if (!(filePath && typeof filePath === "string")) {
-    throw new TypeError("must initialize kzg with the filePath to a txt/json trusted setup");
-  }
-  if (!fs.existsSync(filePath)) {
-    throw new Error(`no trusted setup found: ${filePath}`);
+/**
+ * Gets location for trusted setup file. Uses user provided location first. If
+ * one is not provided then defaults to the official Ethereum mainnet setup from
+ * the kzg ceremony.
+ *
+ * @param {string} filePath - User provided filePath to check for trusted setup
+ *
+ * @returns {string} - Location of a trusted setup file. Validity is checked by
+ *                     the native bindings.loadTrustedSetup
+ *
+ * @throws {TypeError} - Invalid file type
+ * @throws {Error} - Invalid location or no default trusted setup found
+ *
+ * @remarks - This function is only exported for testing purposes.  It should
```

Comment:
Double space between sentences should be removed.
```suggestion
 * @remarks - This function is only exported for testing purposes. It should
```

Reply:
Updated.